contrib/test.sh

```diff
@@ -78,6 +78,9 @@ if [ "$DO_ASAN" = true ]; then
     cargo run --release --manifest-path=./no_std_test/Cargo.toml | grep -q "Verified Successfully"
 fi
 
+# Test if panic in C code aborts the process (we need to invert the exit code)
+! cargo test -- --include-ignored --exact 'tests::test_panic_raw_ctx_should_terminate_abnormally'
+
```

Comment:
This currently fails compiling on stable/1.29.0
```
     Running `/home/runner/work/rust-secp256k1/rust-secp256k1/target/debug/deps/secp256k1-a3ae915be309b8a2 --include-ignored --exact 'tests::test_panic_raw_ctx_should_terminate_abnormally'`
error: The "include-ignored" flag is only accepted on the nightly compiler with -Z unstable-options
```

Reply:
This should work on all of them: `cargo +1.29.0 test -- --ignored --exact 'tests::test_panic_raw_ctx_should_terminate_abnormally'`.

Reply:
(I'd suggest we check for SIGILL/SIGABRT exit codes, but on stable/1.29 it will just panic and exit with 101, the same as it exits here for the flag error)